### PR TITLE
added get_creation_time function for gdscript

### PR DESCRIPTION
**Pull request by notwarp**
It should be improved

### CODE REVIEW

File: drivers/unix/dir_access_unix.cpp [hunk 1]
```diff
@@ -124,6 +124,26 @@ uint64_t DirAccessUnix::get_modified_time(String p_file) {
 	return 0;
 };
 
+// NEW FUNCTION
```
Comment (hpvb) [inline drivers/unix/dir_access_unix.cpp]: These comments really are not necessary

Comment (notwarp) [reply drivers/unix/dir_access_unix.cpp]: i know, it was only for mark the changes i have made. sorry for that

**Comment by mhilbrunner** [comment]
Please update this to fix the CI errors and remove those unnecessary comments :)

**Comment by akien-mga** [comment]
Bump, we can't merge it as it is now.

**Comment by reduz** [comment]
Sorry, merged this one by accident, though i was looking at another one. What is needed to merge this, remove the NEW FUNCTION comments?

**Comment by akien-mga** [comment]
@reduz You could have kept it merged and just edited the comments away :)

**Pull request merged**